src/parsers/in.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+
+import P from 'parsimmon'
+import crap from './crap'
+
+import type { TupleNodeType, PrimitiveNodeType } from '../types'
+
+import StringParser from './primitives/string'
+import ObjectParser from './object'
+
+export default P.seq(
+  StringParser.trim(crap),
+  P.regexp(/\s*in\s*/).trim(crap),
+  ObjectParser.trim(crap).map(
```

Comment:
Here it'd be also better to accept more stuff. It could be a variable, or for example an array/list instead of an object. For now it's enough to give runtime errors here, it's not necessary to enforce type in the parser here.